### PR TITLE
Task List Tracker

### PR DESCRIPTION
**Pull request by Tomm1128**
No real tests. I got the main deliverables done.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,3 +1,34 @@
...
+    todo.textContent = `${userInput} `
+    todo.appendChild(deleteBtn)
+    tasks.appendChild(todo)
+    deleteBtn.addEventListener("click", handleDelete)
+  })
+
+
+
+})
+
```
Comment (stephenmckeon) [inline src/index.js]:
```suggestion
  //grab user submission into input field 
  submitBtn.addEventListener("click", (event) => {
    event.preventDefault()
    const userInput = handleInput()
    const tasks = document.getElementById("tasks")
    const todo = document.createElement("li")
    const deleteBtn = document.createElement("button")
    deleteBtn.textContent = 'X'
    todo.textContent = `${userInput} `
    todo.appendChild(deleteBtn)
    tasks.appendChild(todo)
    deleteBtn.addEventListener("click", handleDelete)
  })
})

```

File: src/index.js [hunk 2]
```diff
@@ -1,3 +1,34 @@
 document.addEventListener("DOMContentLoaded", () => {
   // your code here
-});
+  // User can type task into input field and click submit
+  // User should then see the string provided appear in the DOm after submit
+  const submitBtn = document.querySelector('input[type="submit"]') 
+
+  const handleInput = () => {
```
Comment (stephenmckeon) [inline src/index.js]: What's a better name for this function?

**Review by stephenmckeon — APPROVED** [review]

File: src/index.js (hunk 2 above)
Comment (Tomm1128) [reply src/index.js]: handleSubmit, handleUserInput, or handleTaskInput? These are the three that pop to my head right now. Let me know what you think.

Comment (stephenmckeon) [reply src/index.js]: Hm, is this "handling" anything? Not really... what's a better verb?

Comment (Tomm1128) [reply src/index.js]: maybe getUserInput? or getTaskInput?

Comment (stephenmckeon) [reply src/index.js]: Yeah there ya go! You had the `input` part spot on, so even `getInput` would suffice. Naming is hard, but very important. Thanks TJ!

Comment (Tomm1128) [reply src/index.js]: Thank you for helping me with it!

**Review by stephenmckeon — APPROVED** [review]
This is seriously impressive. Keep it up dude.

**Review by stephenmckeon — COMMENTED** [review]
I'd love to see a screenshot or video in your PR description!

**Comment by Tomm1128** [comment]
This is before there are tasks:

Task_lister_lite

This is after some tasks have been entered:

Task_lister_lite_with_tasks

**Pull request closed without merging**